src/tests/machine-test.lua

```diff
@@ -218,8 +217,11 @@ end)
 print("\n\ntesting proof after step one")
 do_test("proof check should pass", function(machine)
     local log_type = {}
-    machine:step_uarch(log_type)
-    -- find ram memory range
+    machine:log_uarch_step(log_type)
+
+    -- Dump RAM memory to file, calculate hash of file
```

Comment:
This might be my fault, but we are not dumping things to file anymore because we can read directly from the memory range. Can you fix the comment?